scriptmodules/supplementary/runcommand/runcommand.sh

```diff
@@ -778,6 +784,13 @@ function choose_emulator() {
     local default="$2"
     local cancel="$3"
 
+    local mode_desc="default emulator for "
```

Comment:
Maybe use a more suggestive var name instead of `mode_desc` (copy & paste from `choose_mode` ?)

Reply:
Yeah must be. I'll sort. Thanks.

Reply:
Actually this isn't a copy and paste error. The mode variable in both functions is unrelated to video but how the function behaves depending on the first parameter. So when I added a description var I named it appropriately.

It could be argued in the choose_mode function this variable name is confusing but that can be another PR if needed but I'm not too fussed.

Reply:
And more importantly to note the mode variable predates this PR if

> Looks and works fine. Left a couple of small notes.
> 
> One thing that you may consider adding is replacing the empty `()` - when there's no choice for video/emulator (+ROM) - with `not set` or 'none', it would be similar to the title of the selection screen (i.e. _Currently: (not set)_ when no choice exists).

I will do this. Hopefully there is no emulator ever called "not set" :-)

Reply:
Actually I've changed my mind. At least for now as this adds a heap of extra variables and logic for the menu as we have to handle

```
$emu_sys
$emu_rom
$vid_emu
$vid_rom
$fb_emu
$fb_rom
```

My main motivation with this PR was the variable menu options. I don't disagree this may be better but I think that may be better off in a future change.

Reply:
>  I don't disagree this may be better but I think that may be better off in a future change.

It's mostly cosmetic, so yes, it may be handled better in a future change.